src/prefect/cli/build_register.py

```diff
@@ -705,7 +705,11 @@ def watch_for_changes(
 @click.pass_context
 @handle_terminal_error
 def register(ctx, project, paths, modules, json_paths, names, labels, force, watch):
-    """Register one or more flows into a project."""
+    """Register one or more flows into a project.
+
+    Flows with unchanged metadata will be skipped as registering again will only
+    change the version number.
```

Comment:
maybe worth noting that `force` will override this behavior and guarantee new registration

Reply:
(it is documented later in the help string, so we can live without it)

Reply:
Yeah I had a similar thought. I want to keep the top-level description as simple as possible.